deluge/httpdownloader.py

```diff
@@ -133,7 +133,9 @@ def request_callback(self, response):
                 content_disp = headers.getRawHeaders(b'content-disposition')[0].decode(
                     'utf-8'
                 )
-                content_disp_params = cgi.parse_header(content_disp)[1]
+                message = email.message.Message()
```

Comment:
Why not use `EmailMessage` instead of legacy `Message` class?

Reply:
Okay, thanks. will update.